qiita_db/sql_connection.py

```diff
@@ -545,99 +436,156 @@ def fetchall_with_types(self, sql, sql_args=None):
 
         Notes
         -----
-        from psycopg2 documentation, only variable values should be bound
+        From psycopg2 documentation, only variable values should be bound
         via sql_args, it shouldn't be used to set table or field names. For
         those elements, ordinary string formatting should be used before
         running execute.
         """
         with self._sql_executor(sql, sql_args) as pgcursor:
             result = pgcursor.fetchall()
-            types = {desc[0]: self.TYPE_CODES[desc[1]]
-                     for desc in pgcursor.description}
 
-        return result, types
+        return result
 
-    def fetchone_with_types(self, sql, sql_args=None):
-        """Executes a fetchone SQL query with column information
+    def _check_queue_exists(self, queue_name):
+        if queue_name not in self.queues:
+            raise KeyError("Queue %s does not exist" % queue_name)
```

Comment:
For the sake of clarity with whitespaces, etc you may want to do `"Queue '%s' does not exist"`.